src/seedsigner/gui/screens/tools_screens.py

```diff
@@ -230,7 +316,7 @@ def __post_init__(self):
         # Specify the keys in the keyboard
         self.rows = 1
         self.cols = 4
-        self.key_height = GUIConstants.get_top_nav_title_font_size() + 2 + 2*GUIConstants.EDGE_PADDING
+        self.key_height = GUIConstants.get_top_nav_title_font_size() + GUIConstants.EDGE_PADDING//4 + 2*GUIConstants.EDGE_PADDING
```

Comment:
For these large keyboard keys, I think it makes sense to simplify all this and just the constant that's already defined for button height.